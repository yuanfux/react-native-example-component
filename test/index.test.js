import React from 'react';
import renderer from 'react-test-renderer';
import ExampleComponent from '../src';

describe('<ExampleComponent />', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<ExampleComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
