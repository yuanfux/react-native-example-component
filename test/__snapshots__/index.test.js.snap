// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExampleComponent /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello World!
  </Text>
</View>
`;
